cli/util/upload_bin.go

```diff
@@ -295,7 +295,12 @@ func uploadBlocks(ctx *cli.Context, p *pool.Pool, rpc *rpcclient.Client, signer
 
 					objBytes := bw.Bytes()
 					errRetr := retry(func() error {
-						return uploadObj(ctx.Context, p, signer, acc.PrivateKey().GetScriptHash(), containerID, objBytes, attrs, homomorphicHashingDisabled)
+						resOid, errUpload := uploadObj(ctx.Context, p, signer, acc.PrivateKey().GetScriptHash(), containerID, objBytes, attrs, homomorphicHashingDisabled)
+						if errUpload != nil {
+							return errUpload
+						}
+						fmt.Fprintf(ctx.App.Writer, "Uploaded block %d with object ID: %s\n", blockIndex, resOid.String())
```

Comment:
Let's move it under `--debug, d` flag, these logs are useful. By default we don't want these logs to be enabled, but it's a nice opportunity to have them in debug mode.

Reply:
only for blocks?

Reply:
Removed index files OIDs logging.